### PR TITLE
fix(stepper): fix initialStep input to be used correctly by stepper when set

### PR DESCRIPTION
**Pull request by mivaylo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by mivaylo** [comment]
CDE-158

File: projects/angular/src/accordion/stepper/stepper-panel.ts [hunk 1]
```diff
@@ -20,9 +20,9 @@ import { FormGroupName, NgModelGroup } from '@angular/forms';
 import { Subscription } from 'rxjs';
 import { filter, pairwise, tap } from 'rxjs/operators';
 
+import { ClrCommonStringsService } from '../../utils';
 import { IfExpandService } from '../../utils/conditional/if-expanded.service';
 import { triggerAllFormControlValidation } from '../../utils/forms/validation';
-import { ClrCommonStringsService } from '../../utils/i18n/common-strings.service';
```
Comment (kevinbuhmann) [inline projects/angular/src/accordion/stepper/stepper-panel.ts:25]: Please revert this change.

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]
Please change the PR title and summary to describe the actual bug fix instead of an implementation detail. A "make use of stepper initial step input in stepper model" entry in the changelog won't mean anything to a consumer of the library.
